typescript/test/testdata/texttospeech/src/v1/text_to_speech_client.ts.baseline

```diff
@@ -50,7 +50,7 @@ export interface Callback<
 
 export class TextToSpeechClient {
   /*
-  Service that implements Google Cloud Text-to-Speech API.
+   Service that implements Google Cloud Text-to-Speech API.,
```

Comment:
This is weird, why did it add a comma?

Reply:
I actually see this comma added in all baselines. Seems like something is wrong with a template :)

Reply:
yes, you are right, I directly print an array of comments which should be split and printed, thanks!